tests/Games/NexusMods.Games.TestFramework/AIsolatedGameTest.cs

```diff
@@ -377,6 +379,49 @@ private AbsolutePath GetArchivePath(Hash hash)
         store.TryGetLocation(Connection.Db, hash, null, out var archivePath, out _).Should().BeTrue("Archive should exist");
         return archivePath;
     }
+    
+            /// <summary>
+    /// Uses Verify to validate all the states of the game, previously applied states, etc.
+    /// </summary>
+    protected void LogDiskState(StringBuilder sb, string sectionName, string comments = "", Loadout.ReadOnly[]? loadouts = null)
+    {
+        Logger.LogInformation("Logging State {SectionName}", sectionName);
+        
+        var metadata = GameInstallation.GetMetadata(Connection);
+        sb.AppendLine($"{sectionName}:");
+        if (!string.IsNullOrEmpty(comments))
+            sb.AppendLine(comments);
+        
+        Section("### Initial State", metadata.InitialDiskStateTransaction);
+        if (metadata.Contains(GameInstallMetadata.LastSyncedLoadoutTransaction)) 
+            Section("### Last Synced State", metadata.LastSyncedLoadoutTransaction);
+        Section("### Current State", metadata.LastScannedDiskStateTransaction);
+        foreach (var loadout in loadouts ?? [])
```

Comment:
When using Verify, it's important that the output is consistent. If the sorting is indeterminate, you might get inconsistent outputs. I'd suggest sorting by name or something.